### DISCUSSION TITLE
Ozone for my work for creating volume , buckets and storing data without using s3 and Aws through Python3

### DISCUSSION THREAD

**Discussion by FaheemKhan0817** [apache/ozone#6665] [FAQ] [answered]
Can i used stand-alone Ozone for my work for creating volume , buckets and storing data without using s3 and Aws through Python3 script?

**Comment by ChenSammi** [comment]
Yes, Ozone not only supports S3 compatible access(through s3g), but aslo supports Hadoop compatible filesystem API(through ozone java client), and Ozone fs CLI (usually for administrators).

**Reply by FaheemKhan0817** [reply]
how to use python for creating volumes, buckets and store the data in bucket on Ozone **without using AWS and s3**

**Reply by errose28** [reply, marked as answer]
Ozone does not provide its own Python client, only a Java client. The HDFS Java client can also be used. Using S3 protocol with a Python S3 client is the recommended way to interact directly with Ozone through Python.
